frontend/src/hooks/useSearchbar.ts

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
what's the value of placing this hook in a centralised `hooks` folder when it's only ever going to be used in the SearchBar component? or is it actually reusable?

Reply:
Technically you require it to use the searchbar, but the searchbar component itself does not use it. I'll move it into the component folder anyways and export it!